packages/discovery-provider/Dockerfile.prod

```diff
@@ -24,8 +24,8 @@ RUN apk update && \
   linux-headers \
   nodejs \
   npm \
-  python3=3.11.6-r0 \
-  python3-dev=3.11.6-r0 \
+  python3=3.11.8-r0 \
```

Comment:
we should update here too https://github.com/AudiusProject/audius-protocol/blob/main/.python-version-dev